client/layout/masterbar/logged-out.jsx

```diff
@@ -129,9 +121,11 @@ class MasterbarLoggedOut extends React.Component {
 				 */
 				signupUrl = currentQuery.redirect_to;
 			} else {
+				// TODO: should it be changed to '/start' or '/jetpack/connect'?
```

Comment:
Since we ended up here at `/log-in/jetpack` (Jetpack's login view), I'd assume we need `/jetpack/connect`.

Best to confirm by testing Jetpack connect flow with https://jurassic.ninja/

Reply:
Changed to `/jetpack/connect`